jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/jdbc/JdbcMetadataReader.java

```diff
@@ -252,7 +252,7 @@ private static String findSchema(final String schema, final DatabaseMetaData dbM
         final List<SqlDialect.MappedTable> tablesMapped = new ArrayList<>();
```

Comment:
`java.Sql.ResultSet` implements `AutoClosable` use try-with-resources and remove manual `close()`.

Reply:
done